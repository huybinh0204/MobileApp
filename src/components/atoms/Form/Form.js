import PropTypes from 'prop-types';
import React from 'react';
import { FormProvider } from 'react-hook-form';

const Form = ({ children, formMethods }) => {
  return <FormProvider {...formMethods}>{children}</FormProvider>;
};

Form.propTypes = {
  onSubmit: PropTypes.func,
};

export default Form;
