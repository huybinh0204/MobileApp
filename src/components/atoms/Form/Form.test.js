import React from 'react';
import { render } from '@testing-library/react-native';
import Theme from '_utilities/Theme';
import Form from './Form';

describe('From', () => {
  let props;

  const RenderForm = (innerProps) => (
    <Theme>
      <Form {...innerProps} />
    </Theme>
  );

  beforeEach(() => {
    props = {};
  });

  describe('structure', () => {
    it('should match the snapshot', () => {
      const { toJSON } = render(<RenderForm {...props} />);
      expect(toJSON()).toMatchSnapshot();
    });
  });
});
