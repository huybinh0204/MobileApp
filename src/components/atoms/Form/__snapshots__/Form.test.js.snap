// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`From structure should match the snapshot 1`] = `null`;
